assets/html/templates/site-header.php

```diff
@@ -9,7 +9,8 @@
 
 <header class="wrap wrap--header wrap--header__menu-hidden" [class]="mobileMenu ? 'wrap wrap--header' : 'wrap wrap--header wrap--header__menu-hidden'">
 
-	<div class="wrap__item wrap__item--branding">
+	<?php $wrap_item_modifier = isset( $has_separator ) ? '' : 'wrap__item--no-separator'; ?>
```

Comment:
@delawski in what instances is the separator present and not present?

Reply:
@kopepasah On some pages like `category.php`, `archive.php` or `page.php` you'll find a horizontal separator between the branding and the article stream / sidebar.

On others, like homepage or `single.php` there's no separator since article an image is visually separating the branding from the main content.

I suggest checking out designs in Figma while integrating with the back-end to make sure on which page the separator is present and where it's not.

Reply:
If the separator is specific to those specific views, normally it might be better to use the WP body classes for those views to add the bottom border, rather than BE having to add conditionals to add a class to the header. However, since the size of the stylesheet is limited, it might be better for BE to implement the conditionals within the header.

Thoughts?

Reply:
I'd say it also depends... The separator should be present if a post has no featured image. So it's not only a thing that depends on the body class, it's also content-specific.

In my opinion it's not a big deal to have that logic in the site header since it already has page-specific logic. E.g. in the site header a title tag is different on the homepage and on all other pages (`h1` is used on the homepage; other pages use just a plain text).

Reply:
Sounds good to me.